### ISSUE TITLE
HF cache is too large

### ISSUE THREAD

**Issue by DNGros** [DaiseyCode/lmwrapper#4]
It is using like 0.5 GB per token. Working theory is because it is serializing all the logits for every token
